satellite/satellitedb/repairqueue.go

```diff
@@ -0,0 +1,80 @@
+// Copyright (C) 2018 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package satellitedb
+
+import (
+	"context"
+
+	"github.com/golang/protobuf/proto"
+	"storj.io/storj/pkg/pb"
+	"storj.io/storj/pkg/utils"
+	dbx "storj.io/storj/satellite/satellitedb/dbx"
+)
+
+type repairQueueDB struct {
+	db *dbx.DB
+}
+
+func (r *repairQueueDB) Enqueue(ctx context.Context, seg *pb.InjuredSegment) error {
```

Comment:
To avoid passing context to method take a look here https://github.com/storj/storj/blob/queue-dbx/satellite/satellitedb/overlaycache.go#L18